docs: update for v36 (#23161)
renovatebot/renovate: docs/usage/getting-started/running.md

Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
These Docker images work on all the hosting platforms that Renovate supports.

> Both `linux/amd64` and `linux/arm64` architectures are supported, although you may still find some bugs in the `arm64` image.

Reviewer: ```suggestion
Both `linux/amd64` and `linux/arm64` architectures are supported, but you may still find some bugs in the `arm64` image.
```

You can't run the Docker images in a Windows or macOS container.

In general, you can run Renovate natively on Windows as long as you have all tools it will need (e.g. `npm`, `pipenv`, etc.) preinstalled before you run Renovate.

There are two Docker image flavors:

- The default image, which installs required tools at runtime (default for `latest` tag),
- The `-full` image, which comes with latest or very recent versions of every tool pre-installed

> ##### The default image (formerly `slim`)

Reviewer: Your changing the headings in this document, did you check if we have any links that need to be updated?

The default image (also aliased as `-slim`) only comes with the Node.js environment.

> Renovate will then install the required tools at runtime when needed (see documentation for (see documentation for `binarySource=install` for further details).

Reviewer: ```suggestion
Renovate will then install any needed tools when it runs.
Read the `binarySource=install` documentation for more details.
```
